pkgs/rpm/clapper.spec

```diff
@@ -0,0 +1,64 @@
+#
+# spec file for package clapper
+#
+# Copyright (C) 2020  sp1rit
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <https://www.gnu.org/licenses/>.
+
+
+Name:				clapper
+Version:			0.0.0
+Release:			0
+Summary:			A GNOME media player
+License:			GPL-3.0
+URL:				https://github.com/Rafostar/clapper
+BuildArchitectures:		noarch
+BuildRoot:			%{_builddir}/%{name}-%{version}-build
+BuildRequires:			meson gjs
+Requires:			gjs gstreamer
```

Comment:
@sp1ritCS 
Like I noticed earlier `gstreamer` on Fedora/RHEL would install wrong (old) gstreamer-0.10. You should probably move it to `suse_version` and add `gstreamer1` inside the fedora/rhel. This is what Totem does.